File: Modul 4/PRAK402/PRAK302-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main(){
    int inputAngka; int i;
    scanf("%d", &inputAngka);

    for (i = 1; i <= inputAngka; i++){
        if (i % 2 == 1){
             printf("%d ", i);
        } 
        
        if (i == inputAngka){
            printf("\n");
        }
    }


    for (int g = inputAngka; g > 1; g--){
        if (g % 2 == 0){
            printf("%d ", g);
        }
    }
    
    return 0;
}

File: Modul 4/PRAK402/PRAK302-2310817310011-MuhammadRizky.py
inputAngka = int(input())

for i in range(inputAngka + 1):
    if (i % 2 == 1):
        print(i, end=' ')

print("")
g = inputAngka

while g > 0 :
    if (g % 2 == 0) :
        print(g, end=' ')
        g -= 2
    else : g -= 1